src/Controls/src/Core/ImageButton/ImageButton.iOS.cs

```diff
@@ -0,0 +1,46 @@
+#nullable disable
```

Comment:
I agree with @MartyIX 

This is a new file and it would be pretty easy to just remove this directive